message-index/messages/GHC-69158/index.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Conflicting exports
+summary: Different identifiers with the same name are (re-)exported from the same module
+introduced: 9.6.1
+severity: error
+---
+
+This error means that a module is trying to export two identifiers with the same
+name, but which are actually different because they were imported from separate
```

Comment:
```suggestion
name, but which are actually different because they were imported from separate qualified
```

Reply:
It's not necessary for both modules to be qualified; the error can also happen when one of them has unqualified imports, e.g.
```hs
module Foo (Bar.f, module Baz) where

import qualified Bar (f)
import Baz (f)
```
(Taken from the comment on `TcRnConflictingIMports`.)